lib/bemxjst/tree.js

```diff
@@ -353,6 +353,10 @@ Tree.prototype.body = function() {
 
 Tree.prototype.match = function() {
   var children = new Array(arguments.length);
+
+  if (!arguments.length)
+    throw new Error('.match() must have argument');
```

Comment:
`if (!something) throw new Error('error')` is a pure assert pattern: `assert(something, 'error')`
prob we can get some bytes on it.